geoh5py/objects/object_base.py

```diff
@@ -354,6 +335,34 @@ def create_property_group(
 
         return prop_group
 
+    @property
+    def entity_type(self) -> ObjectType:
+        """
+        :obj:`~geoh5py.shared.entity_type.EntityType`: Object type.
+        """
+        return self._entity_type
+
+    @property
+    def extent(self) -> np.ndarray | None:
+        """
+        Geography bounding box of the object.
+
+        :return: Bounding box defined by the bottom South-West and
+            top North-East coordinates,  shape(2, 3).
+        """
+        if self.locations is None:
+            return None
+
+        return np.c_[self.locations.min(axis=0), self.locations.max(axis=0)].T
+
+    @property
+    def faces(self) -> np.ndarray:
+        """Object faces."""
+        # todo: this is used nowhere... remove?
```

Comment:
Not sure, will need to ask Julien. Might just not be supported by our API... It is technically an accepted association type, so something should be using it.